apps/io-services-cms-webapp/src/main.ts

```diff
@@ -11,3 +18,15 @@ export const httpEntryPoint = pipe(
   createWebServer,
   expressToAzureFunction
 );
+
+const config = getConfigOrThrow();
+
+export const queueEntryPoint = handleRequestReview(
+  getDao(config),
+  ServiceReviewProxy(JiraAPIClient(config)),
+  apimProxy(
+    getApimClient(config, config.AZURE_SUBSCRIPTION_ID),
+    config.AZURE_APIM_RESOURCE_GROUP,
+    config.AZURE_APIM
+  )
+);
```

Comment:
Naming:
* `queueEntryPoint` -> Should we have a more descriptive name? Try to think of this name as an exposed API of a module a user will use.
* `handleRequestReview` --> actually, this is a factory method. Use a name which is something like `make___`, `create___` or `build___`